app/src/main/java/lt/vilnius/tvarkau/views/adapters/ProblemsListAdapter.java

```diff
@@ -52,25 +52,21 @@ public void onBindViewHolder(final ViewHolder holder, int position) {
         holder.statusView.setBackgroundColor(context.getResources().getColor(item.getColor()));
         holder.timeView.setText(DateUtils.getRelativeTimeSpanString(item.updatedAt.getTime()));
 
+        String thumbUrl = item.getThumbUrl();
+
+        if (thumbUrl == null) {
+            holder.thumbView.setImageResource(R.drawable.ic_placeholder_list_of_reports);
+        } else {
+            Picasso.with(context).load(thumbUrl).into(holder.thumbView);
+        }
+
         holder.content.setOnClickListener(new View.OnClickListener() {
```

Comment:
RecyclerView don't have 'setOnItemClickedListener' method. It's not really hard to create that. Do you really think it's worth creating ?